src/components/Header.tsx

```diff
@@ -1,31 +1,164 @@
+/* This example requires Tailwind CSS v2.0+ */
+import { Fragment } from 'react';
+import { Disclosure, Menu, Transition } from '@headlessui/react';
+import { BellIcon, MenuIcon, XIcon } from '@heroicons/react/outline';
```

Comment:
Import these icons from `react-icons`. You can find their names here https://react-icons.github.io/react-icons/icons?name=hi
```suggestion
import { HiBell, HiMenu, HiX } from 'react-icons/hi';
```